### PR TITLE
[win10] update libxml2 uwp patch

### PR DESCRIPTION
**Pull request by afedchin**
update because of https://github.com/xbmc/xbmc/pull/13849

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@ksooo @Rechi can we merge this?

**Review by Rechi — APPROVED** [review]

File: depends/windowsstore/libxml2/02-fix-uwp-api.patch [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+-  _CRT_STDC_NO_DEPRECATE)
++  _CRT_SECURE_NO_DEPRECATE
++  _CRT_STDC_NO_DEPRECATE
++  _CRT_NONSTDC_NO_DEPRECATE
++  HAVE_WIN32_THREADS
++  _WINSOCK_DEPRECATED_NO_WARNINGS
++)
+ 
+ if(BUILD_SHARED_LIBS AND MSVC)
+   target_link_libraries(libxml2 PRIVATE ws2_32.lib)
```
Comment (Rechi) [inline depends/windowsstore/libxml2/02-fix-uwp-api.patch:18]: @afedchin applying this hunk of the patch fails. I've no idea where you got that from.

Comment (afedchin) [reply depends/windowsstore/libxml2/02-fix-uwp-api.patch:18]: It worked well 3 months ago. What was changed?

Comment (Rechi) [reply depends/windowsstore/libxml2/02-fix-uwp-api.patch:18]: No, it never applied. Before https://github.com/xbmc/xbmc/pull/15321 failing patches were just ignored.

**Pull request merged**